### ISSUE TITLE
Academy Overview

### ISSUE THREAD

**Issue by zramsay** [cosmos/cosmos-academy#31]
following discussion with @fedekunze @Chjango @gamarin2 we will be organizing the academy and docs in the in the following way:

- The single point of entry to the Tendermint/Cosmos ecosystem for new users will be the home page of: cosmos-academy.readthedocs.io (gitbooks be gone)
- This page will orient users into several streams (e.g., start/join testnet, build an SDK app/module, write an ABCI app/server, using Tendermint Core, etc)
- The rest of Cosmos Academy will give high level overview of concepts, define key terms (glossary) and organize links to all external (e.g., courses by Blockgeeks or whatever) or internal (video tutorial) educational resources.
- Code specific tutorials and (most) examples should be held in the specific repo's Read The Docs page. This ensures (that the majority of ...) breaking changes in code/docs does not require updating the academy.
- Basically, the academy is like a class/textbook that once read you don't really need to refer back to. Docs in the repo are the practical "lab" aspects of the learning.

The line between what goes in 1) the academy, 2) read the docs, and 3) blog posts on medium is hopefully somewhat less blurred with this approach.
